src/Microsoft.ML.Core/Data/ITransformModel.cs

```diff
@@ -25,6 +25,10 @@ public interface ITransformModel
         /// </summary>
         ISchema InputSchema { get; }
 
+        /// <summary>
+        /// Schema of the transform model.
```

Comment:
>Schema of the transform model. [](start = 12, length = 30)

This comment, even if the thing were fixed to return an `ISchema`, is rather odd because models don't *have* schema. They only have schema once applied to model. You will need something like the above description of `InputSchema`, that clarifies the status of this. "Schema of the transform model" is not a meaningful statement.

Reply:
You'll also probably want to name it `OutputSchema`, since it serves a similar purpose to `InputSchema`.

---
In reply to: [190936953](https://github.com/dotnet/machinelearning/pull/239#discussion_r190936953) [](ancestors = 190936953)